player/mpv.go

```diff
@@ -35,6 +35,8 @@ type MPV struct {
 
 var logLibMPV = flag.Bool("log-libmpv", false, "log output of libmpv")
 
+const PROP_PAUSE uint64 = 0
```

Comment:
The proper type for this constant is `C.uint64_t`. With that change, we should be able to remove the casts.